src/gui/application.cpp

```diff
@@ -263,14 +263,11 @@ Application::Application(int &argc, char **argv)
 
 #ifdef PLUGINDIR
     // Setup extra plugin search path
-    QString extraPluginPath = QStringLiteral(PLUGINDIR);
-    if (!extraPluginPath.isEmpty()) {
-        if (QDir::isRelativePath(extraPluginPath))
```

Comment:
Any idea why this was made explicitly absolute here?

Reply:
We used a relative path on Mac, I'm not sure how relative plugin paths are defined(what are they relative to).